me.id.webverify/.gitignore

```diff
@@ -1,6 +1,5 @@
 .gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
+/.idea
```

Comment:
Yes we should. They provide no useful information to developers and occassionally cause conflicts.